jquery.offcanvas.js

```diff
@@ -0,0 +1,50 @@
+;(function (window, document, $) {
+  // Set the negative margin on the top menu for slide-menu pages
+  var $selector1 = $('#topMenu'),
+    events = 'click.fndtn';
+  if ($selector1.length > 0) $selector1.css("margin-top", $selector1.height() * -1);
```

Comment:
Line is too long.<br>Expected '{' and instead saw '$selector1'.<br>Mixed double and single quotes.